doxygen/lang/120_modules.dox.tmpl

```diff
@@ -19,6 +19,8 @@
 %requires ../../directory/Module.qm
     @endcode \n
 
+    Module implementation could be separated to multiple files and stored as a directory named with the same name as the module. To require them use @ref requires "%requires ModuleName" without ".qm" extension. All files inside the directory will be automatically included at program parse time.
+
```

Comment:
please enrich the documentation (there are only some extensions allowed), also include some example e.g. dir tree (there must be .qm file with header, then there can be others, etc...)